spec/openapi.yaml

```diff
@@ -2518,6 +2537,15 @@ components:
             discountcode:
               description: Discountcode
               type: string
+            business_model:
+              description: Which business model should be applied to the order. For `B2B` the `company`
+                field must not be empty. For `B2C` the `reference` field is not available. If this field is omitted
+                the store setting will be used, or `B2C` if the store setting is `ALL`
```

Comment:
> If this field is omitted the store setting will be used

This means that the `company` field is required when `B2B` is chosen as default, which wasn't required before. Perhaps this detail should be explained a bit more explicitly because it is a BC break?